.github/workflows/cypress.yaml

```diff
@@ -89,3 +117,8 @@ jobs:
         with:
           name: cypress-videos
           path: cypress/videos
+
+      - name: Delete build artifact
+        uses: actions/delete-artifact@v3
+        with:
+          name: build
```

Comment:
there will be a race condition if one of the parallel job deletes this artifact, so this should be moved to a new job dependent on the parallel-run job